submissions/kotlyar-andrey/dom/style.css

```diff
@@ -0,0 +1,345 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+html,
+body {
+  height: 100%;
+  background-color: #212121;
+}
+@font-face {
+  font-family: darksouls;
+  src: url(fonts/CormorantUnicase-Regular.ttf);
```

Comment:
Avoid using `.ttf` fonts in the web use `woff2` along with `.woff` Use [this converter](https://transfonter.org/) for it